frameworks/JavaScript/nodejs/handlers/mysql-raw.js

```diff
@@ -21,6 +26,16 @@ const queries = {
   }
 };
 
+const populateCache = (callback) => {
+  if (cachePopulated) return callback();
+  connection.query(queries.ALL_WORLDS, (err, rows) => {
```

Comment:
I think we settled on just the regular world table. Will update the specifications soon. Still a work in progress.